3rdparty/node-maintenance-operator/go.mod

```
@@ -0,0 +1,25 @@
+module github.com/edgelesssys/constellation/v2/operators/constellation-node-operator/v2/3rdparty/node-maintenance-operator
```

```suggestion
module github.com/edgelesssys/constellation/v2/3rdparty/node-maintenance-operator
```
I would use the real path for the module name, everything else is confusing.